README.md

```diff
@@ -36,6 +36,22 @@ where:
         -s/--source-dir: The name of the source directory.
         Default: <plugin_name>_<plugin_type>
 
+        -pd/--platform-dependencies: The platform dependencies for the Agent Plugin.
+        Options:
+        common: All dependencies are packaged once, and shared across all platforms. This
+            should only be used if all dependencies are known to be platform-independent.
+        separate: Some or all dependencies are platform-dependent, and are therefore packaged
+            separately for each supported platform. This is the most reliable option,
+            however it results in a larger plugin file, since dependencies are
+            duplicated for each platform.
+        autodetect: The plugin builder will attempt to detect the best method to use.
+        Default: seperate
```

Comment:
Maybe this isn't a question for this PR, by why isn't the default `autodetect`?

Reply:
@cakekoa ?